### PR TITLE
Fit covid and norovirus to Rothman MGS

### LINKED ISSUE #57: Add tests for stats.py
Probably should wait until we have the model/interface more developed.

### LINKED ISSUE #106: Stats: run model for more viruses
Expand the glue code to work more generally. At least for norovirus, possibly for all DNA viruses.

### PR DESCRIPTION
**Pull request by dp-rice**
Refactor and generalize the fitting code to work with covid and norovirus incidence data and Rothman MGS (partially resolves #106). Also add some tests for `stats.py` (resolves #57).

Removes `fit_covid.py`. Now all fitting is done by `fit_rothman.py` which outputs `tsv` files with posterior samples (not in the repo) and prints:
```
$  ./fit_rothman.py 2> /dev/null

-------------------------------------------
sars_cov_2 relative abundance per incidence
-------------------------------------------
Posterior arithmetic mean:
6.0e-12
Posterior geometric mean:
5.8e-12
Posterior quantiles:
     5%        25%        50%        75%        95%
4.0e-12    5.0e-12    5.8e-12    6.7e-12    8.5e-12


------------------------------------------
norovirus relative abundance per incidence
------------------------------------------
Posterior arithmetic mean:
1.1e-08
Posterior geometric mean:
1.1e-08
Posterior quantiles:
     5%        25%        50%        75%        95%
8.2e-09    9.7e-09    1.1e-08    1.3e-08    1.5e-08
```

### CODE REVIEW

**Comment by dp-rice** [comment]
@jeffkaufman, only if you have time before you go.

Also, I added and removed a large binary file from the repo. I hope that that doesn't make things too slow since it shouldn't show up in any diffs if I squash and merge.

File: fit_rothman.py [hunk 1]
```diff
@@ -2,19 +2,18 @@
...
 
 def geom_mean(x: np.ndarray) -> float:
     return np.exp(np.mean(np.log(x)))
 
 
-def print_summary(
-    pathogen: str, naive_ra_per100: float, model_ra_per100: np.ndarray
-) -> None:
+def print_summary(pathogen: str, model_ra_per100: np.ndarray) -> None:
     title = f"{pathogen} relative abundance at 1% prevalence"
```
Comment (jeffkaufman) [inline fit_rothman.py:17]: I think the at `1% prevalence` is now wrong in cases where we're working on incidence?

File: stats.py [hunk 2]
```diff
@@ -1,68 +1,144 @@
...
+        f = lambda i: getattr(attrs[i - 1], attr)
+        return np.vectorize(f)
+
+    df["date"] = get_sample_attrs("date")(df["sample"])
+    df["county"] = get_sample_attrs("county")(df["sample"])
+    df["plant"] = get_sample_attrs("fine_location")(df["sample"])
+    df["total_reads"] = get_sample_attrs("reads")(df["sample"])
+
+    df["viral_reads"] = df["y_tilde"]
+    df["incidence_per100k"] = np.exp(df["theta"])
```
Comment (jeffkaufman) [inline stats.py:69]: I thought the model was going to be agnostic on whether it was being given an incidence or a prevalence?  (here and below)

**Review by jeffkaufman — CHANGES_REQUESTED** [review]
just minor stuff

**Comment by dp-rice** [comment]
@jeffkaufman if you have time before you go, could you please take a look at the (small) change I made to `pathogen_properties.py`?

**Comment by jeffkaufman** [comment]
Change to pathogen_properties LGTM!

**Pull request merged**